### PR TITLE
Update dataset usage and to version 0.11 for Sentiment Analysis 

### PR DESCRIPTION
**Pull request by JRAlexander**
Revising with UCI Sentiment Dataset for greater model accuracy, and updating to 0.11.0.

### CODE REVIEW

**Comment by prathyusha12345** [comment]
@JRAlexander I tested the PR.working fine. But I have questions on the output. 
1.The accuracy metrics is 79%. - I believe accuracy should be more than 85%.Correct me if I am wrong.
2. See the probabilities in below image. how come a text with probability <50% is showing prediction correctly?
image

I believe that probability also should be > 50% to get prediction correctly. Correct me if I am wrong.

@CESARDELATORRE Adding Cesar for reference.

**Comment by JRAlexander** [comment]
It leaves the door open for improving the model. This is the dataset we are using. It has better accuracy than using wikidetox.

**Comment by CESARDELATORRE** [comment]
Well, a rule of thumb is to have an accuracy higher than 80%. However, depending on the business/domain it might need to be a lot higher or even be acceptable to be lower.

When using small datasets it is normal in most cases that the accuracy won't be very super high, neither the probabilities.

Since we're using small datasets for short trainings, I'd suggest not to show the probability in the code or output..

**Comment by JRAlexander** [comment]
I'm going to leave it in there with an explanation about the small data

File: machine-learning/tutorials/SentimentAnalysis/Program.cs [hunk 1]
```diff
@@ -134,117 +128,120 @@ public static void Evaluate(MLContext mlContext, ITransformer model)
...
                 {
-                    SentimentText = "I love this article."
+                    SentimentText = "I love this spaghetti."
                 }
             };
-            // </Snippet26>
+            // </SnippetCreateTestIssues>
 
-            // <Snippet27>
+            // <SnippetLoadModel>
```
Comment (luisquintanilla) [inline machine-learning/tutorials/SentimentAnalysis/Program.cs]: For consistency with SaveModelAsFile, consider moving this logic into its own method.

Comment (JRAlexander) [reply machine-learning/tutorials/SentimentAnalysis/Program.cs]: Ok, that will be a good rev. I'll add an issue to track it. Would rather go ahead and get it out.

Comment (luisquintanilla) [reply machine-learning/tutorials/SentimentAnalysis/Program.cs]: Sounds good.

**Comment by JRAlexander** [comment]
Added your suggestion as issue here https://github.com/dotnet/docs/issues/11221

**Comment by JRAlexander** [comment]
Thanks, @luisquintanilla!

**Pull request merged**